packages/webapp/src/components/Inputs/DateTime/styles.module.scss

```diff
@@ -0,0 +1,53 @@
+/*
+ *  Copyright 2025 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+// styles taken from the <Input /> component
+// !important was necessary as the MUI-applied classes have quite high specificity
```

Comment:
We could either increase the specificity of the selectors here, or use MUI's `sx` prop to customize what we want to override -- important is almost never the answer unless truly as a last desperate resort 😅

Reply:
It's okay! Much better than important, thanks for updating ☺️